site-src/guides/index.md

```diff
@@ -58,9 +63,67 @@ documentation](/concepts/versioning/).
 To install the experimental channel, run the following kubectl command:
 
 ```bash
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/experimental-install.yaml
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/experimental-install.yaml
 ```
 
+### v1.1 Upgrade Notes
+If you are already using previous versions of GRPCRoute or BackendTLSPolicy
+experimental channel CRDs from previous Gateway API releases, you'll need to be
+careful with this upgrade. If you haven't installed Gateway API before, or have
+exclusively used the standard channel of the API, you can skip the rest of this
+section.
+
+#### GRPCRoute
+**Summary:** If you're already using v1alpha2 GRPCRoute, stick with the
+experimental channel of GRPCRoute in v1.1 until the implementation(s) you're
+relying on have been updated to support GRPCRoute v1.
+
+**Explanation:** With the graduation of GRPCRoute to GA, it is now included in
+standard channel. Unfortunately, that can be problematic for anyone that was
+already using the experimental channel version of GRPCRoute. As a rule, CRDs in
+standard channel do not expose alpha API version to avoid any version
+deprecations in that channel. That means that the standard channel version of
+GRPCRoute excludes v1alpha2. All implementations of GRPCRoute built before the
+v1.1 release of Gateway API would have exclusively relied on v1alpha2 and will
+need to be updated to support GRPCRoute v1. Until implementations have been
+updated to support v1, you can safely upgrade to the experimental channel
+version of GRPCRoute included in v1.1 that exposes both v1 and v1alpha2.
+
+**Upgrade Sequence:** If you're already using v1alpha2 GRPCRoute, we'd recommend
+the following upgrade sequence:
+
+1. Install *experimental* v1.1 GRPCRoute CRD
+2. Update all your manifests to use `v1` instead of `v1alpha2`
+3. Upgrade to an implementation that supports GRPCRoute `v1` API Version
+4. Install *standard* channel v1.1 GRPCRoute CRD
```

Comment:
Should we mention as point 5 to re-apply all the manifests that were updated in step 2?

Reply:
Do we need a step to clean up CRDs that are not included in the standard channel?

Reply:
Good questions, re-applying isn't necessary in this case because although the standard channel version of GRPCRoute does not expose v1alpha2 for reads or writes, it does include the schema so it can auto-convert as needed. 

I also don't think we need to cover cleaning up old experimental CRDs since this is entirely focused on GRPCRoute, and the experimental version would be cleaned up when upgrading to standard channel. I think it's still valid to have other resources like BackendTLSPolicy that are only in standard channel, though we admittedly have not been very clear about our policy of mixing and matching resources from different channels...

Open to follow ups on either front, but I think this PR is overall a good enough improvement to get in now and then just iterate as needed from here.